asmcli/commands/install.sh

```diff
@@ -250,3 +250,16 @@ install_control_plane() {
 
   apply_kube_yamls
 }
+
+install_remote_secret() {
```

Comment:
I think this belongs in the `add_to_mesh` file, I don't think it will ever get called in `asmcli install blablabla`

Reply:
I put it here because 1) it has install in its name and 2) we might need it for apply. Should I put this in `util.sh` or just in`add-to-mesh.sh`?

Reply:
For now, in `add-to-mesh`. It might move to `util` later but I'm not 100% sure we'll need it in apply, and I want discrete commits to be consistent.

Reply:
Done